lib/cocoapods/executable.rb

```diff
@@ -123,7 +139,7 @@ def initialize(io = nil)
       #
       # @return [void]
       #
-      def <<(value)
+      def <<(obj)
         super
       ensure
```

Comment:
Perhaps with the current code the ensure itself is pointless – I'll add a test and possibly remove.